src/components/navigation/Navigation.tsx

```diff
@@ -183,7 +182,7 @@ const Navigation = ({ open, width, onNavigationToggle }: NavigationProps) => {
 
                     <HelpMenu />
 
-                    <Box
+                    {/* <Box
```

Comment:
Commented out to not lose the styling that was coded up here. Will be adding toggle back in post mvp.